README.md

```diff
@@ -49,6 +49,7 @@ yarn create nuxt-app <my-project>
     - [Axios](https://github.com/nuxt-community/axios-module)
 3. Choose your favorite UI framework:
     - None (feel free to add one later)
+    - [Framevuerk](https://framevuerk.com)
```

Comment:
Better to put as alphabetical order

Reply:
UI frameworks mentioned in README.md file isn't ordered at all. Where should I put **framevuerk**?

Reply:
OK, I'll do it sepratly